src/App.tsx

```diff
@@ -17,18 +60,34 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                onSelectChange={setSelectedFilter}
+                onInputChange={setInputFilter}
```

Comment:
Consider following the naming conventions for event handler functions. For example, use 'handleInputChange' instead of 'onInputChange'.